sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (Unreleased)
```

Comment:
Please update the version in https://github.com/Azure/azure-sdk-for-net/blob/7ee870d8699fb316c25e595c098ae3c3e59fb074/sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/src/Azure.ResourceManager.RecoveryServicesSiteRecovery.csproj#L3 to align with this changelog entry, otherwise the CI will fail